cloudferrylib/os/actions/snapshots.py

```diff
@@ -27,14 +27,18 @@ def run(self, *args, **kwargs):
         # dump mysql to file
         # probably, we have to choose what databases we have to dump
         # by default we dump all databases
+        user = self.cloud.cloud_config.mysql.user
+        password = self.cloud.cloud_config.mysql.password
+        path = self.cloud.cloud_config.snapshot.snapshot_path
+        if password == '':
+            password_arg = ''
+        else:
+            password_arg = "--password=" + password
```

Comment:
Please keep [DRY](http://en.wikipedia.org/wiki/Don't_repeat_yourself) and move the password/no password logic to a separate method/class.